lib/Makefile

```diff
@@ -27,9 +27,16 @@ MODULE_big = pg_bulkload
 
 EXTENSION = pg_bulkload
 
+# set pg_bulkload version
+BULKLOAD_VERSION = 3.1.20
```

Comment:
Can I get from here?
https://github.com/ossc-db/pg_bulkload/blob/2d9a22578ba57913d8f07960d2815cf93aa277a3/include/common.h#L15

Reply:
I will modify this code.

Reply:
I changed this code.